action.yml

```diff
@@ -2,52 +2,51 @@ name: Update Deployment Board
 description: Adds deployment Issues to an Automated Deployment Project Board for visual tracking of what has been deployed to each environment.
 
 inputs:
-  github-token:
-    description: 'A token with permissions to create and update issues'
+  workflow-actor:
+    description: The GitHub user who triggered the workflow
     required: true
-  github-login:
-    description: 'The login associated with the github-token.  Defaults to github-actions'
+  token:
+    description: A token with permissions to create and update issues
     required: true
   environment:
-    description: 'The environment the branch, tag or SHA was deployed to'
+    description: The environment the branch, tag or SHA was deployed to, i.e. [DEV|QA|STAGE|DEMO|UAT|PROD]
     required: true
-  board-number:
-    description: 'The number of the project board that will be updated.  Found by using the number in the board url.'
+  project-slug:
+    description: The repo the deployment will be created in, i.e. [org|owner]/[repo name]
     required: true
   ref:
-    description: 'The branch, tag or SHA that was deployed'
+    description: The branch, tag or SHA that was deployed
     required: true
-  ref-type:
-    description: 'The type of ref that was deployed  [branch|tag|sha].  If not provided the action will use some regex patterns to identify the type.'
-    required: false
-  deployable-type:
-    description: 'The type of artifact that was deployed like MFE, BFF, API, SVC, DB, etc.  Useful when a release with one tag contains multiple deployable artifacts that are deployed separately.'
-    required: false
-  deploy-status:
-    description: 'The status of the deployment [success|failure|cancelled|skipped]'
+  deployment-status:
+    description: The status of the deployment [error|failure|success]
     required: true
-  deploy-label:
-    description: 'The optional label of the deployment [deleted|destroyed|<your-custom-label>]'
+  deployment-message:
+    description: Any message to be delivered along side of the status
```

Comment:
Does this value go into the `description` property of the Deployment object? Where does it show up on the deployment board? Should we change the description of this input to say where the value is going to be displayed on the deployment board?